app/api/queue.v2/infrastructure/StringJobSerializer.ts

```diff
@@ -0,0 +1,69 @@
+import { Definition, JobSerializer } from '../application/JobSerializer';
+import { Job } from '../contracts/Job';
+
+const DEFAULT_DESCRIPTOR = {
```

Comment:
That's arbitrary. The `Object.create` method uses property descriptors. Unless specified, it will make, between other things, the attributes not enumerable, which implies they are not shown in the console logs. So I just made them enumerable.
I'm anyway considering moving away from this. Options are: 
- Serialize the whole property descriptor. Which would allow to properly replicate them.
- Do not serialize the whole class but only a `data` property.
I'm leaning towards the later because this deserialization does not do deep deserialization. Meaning it won't handle nested class instances, and will just return a plain object. Following the later approach we can say that is a restriction that the `data` needs to be a POJO, otherwise the deserialization might not result as expected.